girder-dandi-archive/girder_dandi_archive/__init__.py

```diff
@@ -19,6 +22,9 @@ def load(self, info):
             {"$setOnInsert": {"value": 0}},
             upsert=True,
         )
+        Setting().set(SettingKey.CORS_ALLOW_ORIGIN, "*")
```

Comment:
Can you delete [this documentation block](https://github.com/dandi/dandiarchive/blob/master/DEPLOYMENT.md#cors), and incorporate the text from there into a code comment here?